File: DART/actividades/actividad 27-05.dart
import 'dart:io';

void main(){
  print('Escriba el valor de  su declaración de renta');
  int n = int.parse(stdin.readLineSync()!);

  if (n <= 10){
    double DRentax = (n * 20/100);
    double DRenta = n - DRentax;
    print('se aplico el descuento y el valor final es : $DRenta\$');
  }else if (n <= 10){
    double DRentax = (n * 10/100);
    double DRenta = n - DRentax;
    print('se aplico el descuento y el valor final es : $DRenta\$');
  }else if (n <= 10){
    double DRentax = (n * 5/100);
    double DRenta = n - DRentax;
    print('se aplico el descuento y el valor final es : $DRenta\$');
  }else {
    double DRentax = (n * 1/100);
    double DRenta = n - DRentax;
    print('se aplico el descuento y el valor final es : $DRenta\$');
  }
}

File: clases y trabajos/RUBY/actividades/actividad 27-05.rb
puts "Escriba el valor de  su declaración de renta"
dr = gets.chomp.to_f

if  dr <= 10
     DRentax = (dr * (20/100))
     DRenta = dr - Drentax
    puts("se aplico el descuento y el valor final es :#{DRenta}$")
elsif dr <= 100
    DRentax = (dr * (10/100))
    DRenta = dr - Drentax
   puts("se aplico el descuento y el valor final es :#{DRenta}$")
elsif dr <= 1000
    DRentax = (dr * (5/100))
    DRenta = dr - Drentax
   puts("se aplico el descuento y el valor final es :#{DRenta}$")
else 
     DRentax = (dr * (1/100))
     DRenta = dr - Drentax
    puts("se aplico el descuento y el valor final es :#{DRenta}$")
end